ui/imodel-components-react/src/test/viewport/ViewportComponentEvents.test.ts

```diff
@@ -16,7 +16,7 @@ import { ViewportComponentEvents } from "../../imodel-components-react/viewport/
 import { TestUtils } from "../TestUtils";
 
 describe("ViewportComponentEvents", () => {
-  const onSelectedViewportChanged = new BeUiEvent();
+  const onSelectedViewportChanged = new UiEvent();
```

Comment:
This one had not changed in 2 years and was already using the correct BeUiEvent ;)

Reply:
Dah... thanks